src/main/resources/templates/languages/cy/non-strategic/astDailyHearingList.json

```diff
@@ -0,0 +1,17 @@
+{
+  "listName": "Rhestr ddyddiol y Tribiwnlys Cefnogi Ceiswyr Lloches",
```

Comment:
Similar comment to frontend regarding double checking welsh includes the change to add hearing

Reply:
Updated